### DISCUSSION TITLE
Can FNF be compiled on windows 7?

### DISCUSSION THREAD

**Discussion by pKunnn** [FunkinCrew/Funkin#1750] [Compiling help] [answered]
**UPDATE: IT WORKS!!**

I was trying to compile the game, i already download everything that requires, but the SDK's that i have to download on visual studio code is a Windows 10 SDK, then after making download of everything i tried compiling the game, and i know that the first time compiling takes a while, but 4gb of ram and 500gb of HD, 2 days is kinda long right? So i may wondering if is impossible to compile fnf in win 7 (And yes, i tried on another pc, not mine pc of course, and it worked, it was a win 10 pc). Thx for reading this and i hope i can get a awnser.

**Comment by libauror-a** [comment]
4gb is really bad for compiling this big of a game

**Reply by ggghhhsssttt** [reply]
it does take alot of time when you first compile, but after you like compile the same game again you get like 20-10 secs of compile time on 4GB Ram PC

**Comment by UNSTOP4BLE** [comment, marked as answer]
compiled it perfectly fine on my 2007 imac win 7, 4gb ram, core 2 duo

**Reply by pKunnn** [reply]
how long it took? i might have downloaded some wrong stuff. going to check that

**Reply by UNSTOP4BLE** [reply]
Like 20-25 min

**Comment by Reycko** [comment]
It should work!

**Comment by isakube** [comment]
i mean good luck but

**Comment by pKunnn** [comment]
Update: it works :+1:

**Comment by ErenKaya31** [comment]
im compiling my source code and it takes forever its bee'd like holy shit 1 day

**Comment by ItzOldFlagDEV** [comment]
I think yes, cuz my friend build kade engine (Windows 7 64bit)
